<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvOptDialog</class>
 <widget class="QWidget" name="ConvOptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="Label9">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>RINEX Version</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="RnxVer">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>2.10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.02</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.03</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Label12">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Station ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="RnxCode">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="RnxFile">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>RINEX Name</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBox1">
     <property name="toolTip">
      <string/>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="Label2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>RunBy/Obsv/Agency</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="RunBy">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="2">
       <widget class="QLineEdit" name="Name0">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="5" colspan="2">
       <widget class="QLineEdit" name="Name1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QLabel" name="Label10">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Comment</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="5">
       <widget class="QLineEdit" name="Comment0">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="5">
       <widget class="QLineEdit" name="Comment1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Maker Name/#/Type</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="Marker">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="2">
       <widget class="QLineEdit" name="MarkerNo">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="5" colspan="2">
       <widget class="QLineEdit" name="MarkerType">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Label4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Rec #/Type/Vers</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="Rec0">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="3" colspan="2">
       <widget class="QLineEdit" name="Rec1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="5" colspan="2">
       <widget class="QLineEdit" name="Rec2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="Ant0">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="3" colspan="2">
       <widget class="QLineEdit" name="Ant1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="5" colspan="2">
       <widget class="QLineEdit" name="Ant2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="Label6">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Approx Pos XYZ</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="AutoPos">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="AppPos0">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>0.0000</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3" colspan="2">
       <widget class="QLineEdit" name="AppPos1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>0.0000</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5" colspan="2">
       <widget class="QLineEdit" name="AppPos2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>0.0000</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="AntDel0">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>0.0000</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3" colspan="2">
       <widget class="QLineEdit" name="AntDel1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>0.0000</string>
        </property>
       </widget>
      </item>
      <item row="7" column="5" colspan="2">
       <widget class="QLineEdit" name="AntDel2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>0.0000</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="ScanObs">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Scan Obs Types</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="2" colspan="2">
       <widget class="QCheckBox" name="OutIono">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Iono Corr</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="4" colspan="2">
       <widget class="QCheckBox" name="OutTime">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Time Corr</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="6">
       <widget class="QCheckBox" name="OutLeaps">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Leap Sec</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="Label7">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Ant Delta H/E/N</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="Label5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Ant #/Type</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="2">
       <widget class="QLineEdit" name="ExSats">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="Label11">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Excluded Satellites</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QGroupBox" name="GroupBox2">
        <property name="toolTip">
         <string/>
        </property>
        <property name="title">
         <string>Satellite Systems</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="Nav1">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>GPS</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Nav2">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>GLO</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Nav3">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>GAL</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Nav4">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>QZS</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Nav5">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>SBS</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Nav6">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>BDS</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Nav7">
           <property name="text">
            <string>IRN</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QGroupBox" name="GroupBox3">
        <property name="toolTip">
         <string/>
        </property>
        <property name="title">
         <string>Observation Types</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="Obs1">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>C</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Obs2">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>L</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Obs3">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>D</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Obs4">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>S</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="GroupBox4">
        <property name="toolTip">
         <string/>
        </property>
        <property name="title">
         <string>Frequencies</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="Freq1">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>L1</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Freq2">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>L2</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Freq3">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>L5/L3</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Freq4">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>L6</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Freq5">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>L7</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Freq6">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>L8</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Freq7">
           <property name="text">
            <string>L9</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnMask">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Mask...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="Label8">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Option</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="RcvOption">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Label3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Debug</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="TraceLevel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>OFF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 5</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnOk">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;OK</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnCancel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>GroupBox1</zorder>
  <zorder>widget</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget_3</zorder>
  <zorder>widget_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
